maintenance/zero_downtime_migration.rst

```diff
@@ -0,0 +1,167 @@
+.. _zero_downtime_migration:
+
+Zero-downtime Migration
+===============================
+
+Firely Server v6 introduces an enhanced MongoDB database migration process. In contrast to the previous migration method that required system downtime, this new approach enables zero-downtime migration. This allows for continuous operation of your existing Firely Server instance while the data migration to the new schema proceeds.
+
+.. note::
+   This migration process applies to the *vonkdata* database. 
```

Comment:
I'm not sure we always reference to this db with the name 'vonkdata'. I would describe it as well, like:

the main database, configured as in :ref:`configure_mongodb`